backports/freeimage.3.8/ENABLE_ON

```diff
@@ -0,0 +1 @@
+v3.8
```

Comment:
```suggestion
v3.7
v3.8
```
and please change directory name to `freeimage.v3.8`.

Reply:
Does it mean that only a `freeimage.v3.8` folder is necessary i.e. should I remove the `freeimage.v3.7`folder?

Reply:
Done in c87975ee5f1bf645c71aa8b0c285461037cafcd4